.gitignore

```diff
@@ -5,3 +5,4 @@
 .DS_Store
 venv/
 .pytest_cache/
+__pycache__/
```

Comment:
please remove pycache from your git repository

Reply:
`__pycache__/` is still not removed, your PR should include only changes in the `main.py` file.